mysql-test/columnstore/basic/t/func_jsonarrayagg.test

```diff
@@ -8,7 +8,9 @@ USE json_arrayagg_db;
 CREATE TABLE t1 (a INT, b INT)ENGINE=COLUMNSTORE;
 INSERT INTO t1 VALUES (1, 1),(2, 1), (1, 1),(2, 1), (3, 2),(2, 2),(2, 2),(2, 2);
 SELECT JSON_VALID(JSON_ARRAYAGG(a)) FROM t1;
+sorted_result;
```

Comment:
This must be prefixed with SQL comment sequence IIRC  -- . Could you double check this?